src/components/cart.js

```diff
@@ -74,12 +75,15 @@ export default class Cart extends Component {
    * @return {String} HTML
    */
   get lineItemsHtml() {
-    return this.model.lineItems.reduce((acc, lineItem) => {
-      const data = merge(lineItem, this.options.viewData);
+    return this.cache.reduce((acc, lineItem) => {
+      const data = Object.assign({}, lineItem, this.options.viewData);
       data.classes = this.classes;
       data.lineItemImage = this.imageForLineItem(data);
       data.variantTitle = data.variant.title === 'Default Title' ? '' : data.variant.title;
-      data.formattedPrice = formatMoney(data.variant.price * data.quantity, this.globalConfig.moneyFormat);
+      if (this.config.moneyFormat) {
+        this.moneyFormat = this.globalConfig.moneyFormat;
```

Comment:
okay yeah, this reads weirdly. it's because `globalConfig.moneyFormat` takes in the value of `config.moneyFormat` and parses it, but if `config.moneyFormat` is null, it sets a default value. what this logic is doing is: "if moneyFormat was explicitly set, use that value, rather than the this.moneyFormat which was set in the init method". I'll change the init assignment to a different variable to make this clearer.